Assets/Scripts/UI/HostLobbyButton.cs

```diff
@@ -0,0 +1,15 @@
+using Lobby;
+using UnityEngine;
+
+namespace UI
+{
+    public class HostLobbyButton : MonoBehaviour, IButtonPressHandler
+    {
+        [SerializeField] private LobbyManager LobbyManager;
+        
+        public void ButtonPressed()
+        {
+            LobbyManager.CreateLobby();
```

Comment:
Does this script need to exist if all it's doing is calling a method from LobbyManager? We could add the function to the button's OnClick event directly from there.